README.md

```diff
@@ -70,6 +70,15 @@ and you can sign in using one of the credentials below:
 
 ```
 
+### E2E Tests
+
+For now, make sure you have your backend and frontend running.
```

Comment:
Since we have two different projects for frontend and backend, I'd like to keep them totally independent. What are your thoughts on mocking cypress requests? https://docs.cypress.io/api/commands/intercept#Syntaxb

Reply:
I was thinking to add that as a future improve, but I can do now